### PR TITLE
Add bootstrap makefile for openshift-ci

### PR DESCRIPTION
**Pull request by ray-harris**

### CODE REVIEW

File: templates/Makefile.build-harness-openshift-ci [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export BUILD_HARNESS_EXTENSIONS_PATH ?= /opt/build-harness/build-harness-extensions
+-include $(BUILD_HARNESS_EXTENSIONS_PATH)/Makefile
+
+.PHONY : init
+## Init build-harness
+init::
+	@echo $(shell date)
+	@echo "No initialization needed for build harness in OpenShift CI"
+
+.PHONY : clean
```
Comment (arewm) [inline templates/Makefile.build-harness-openshift-ci:23]: I would say that clean is not supported... not that people should try to call it a openshift ci job.

File: templates/Makefile.build-harness-openshift-ci [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export BUILD_HARNESS_EXTENSIONS_PROJECT ?= build-harness-extensions
+export BUILD_HARNESS_EXTENSIONS_BRANCH ?= master
+export BUILD_HARNESS_EXTENSIONS_PATH ?= /opt/build-harness/build-harness-extensions
+-include $(BUILD_HARNESS_EXTENSIONS_PATH)/Makefile
+
+.PHONY : init
+## Init build-harness
+init::
+	@echo $(shell date)
+	@echo "No initialization needed for build harness in OpenShift CI"
```
Comment (arewm) [inline templates/Makefile.build-harness-openshift-ci:21]: Can we say that we are using the build harness included in the builder image or something? Or print out the path?

File: templates/Makefile.build-harness-openshift-ci (hunk 1 above)
Comment (ray-harris) [reply templates/Makefile.build-harness-openshift-ci:25]: done

File: templates/Makefile.build-harness-openshift-ci (hunk 2 above)
Comment (ray-harris) [reply templates/Makefile.build-harness-openshift-ci:21]: done

**Pull request merged**